direnv/README.md

```diff
@@ -0,0 +1,25 @@
+# zprezto direnv plugin
+
+## Installation
+
+* Install [`direnv`](https://direnv.net/), probably with
+
+```
+brew install direnv
+```
+
+* Install the contrib repo
```

Comment:
We probably don't need instructions on how to install the contrib repo here since they're already in the main readme.